cardano-cli/src/Cardano/CLI/EraBased/Transaction/Command.hs

```diff
@@ -202,7 +202,6 @@ data TransactionBuildEstimateCmdArgs era = TransactionBuildEstimateCmdArgs
   , scriptFiles :: ![ScriptFile]
   -- ^ Auxiliary scripts
   , metadataFiles :: ![MetadataFile]
-  , mUpdateProposalFile :: !(Maybe (Featured ShelleyToBabbageEra era (Maybe UpdateProposalFile)))
```

Comment:
Need to confirm with @gitmachtl if this is ok.

Reply:
Whats the goal of this experimental api?

Reply:
You addressed my concern here: https://input-output-rnd.slack.com/archives/C031GFLC3DF/p1746552657673919

Reply:
> Whats the goal of this experimental api?
Simplification.

Reply:
Apologies I tagged the wrong Martin. I meant to tag @mkoura .